### DISCUSSION TITLE
Signal Calculation - Developmental branch

### DISCUSSION THREAD

**Discussion by david32** [danielhrisca/asammdf#722] [General]
In the old version, you could calculate signals based on the signals already presented in the plot area. 

In the new version, operations on signals can be performed only on existing signals that can be searched. 
Realized via QLineEdit vs QComboBox.

Was this done on purpose?
Perhaps its better to go back to QComboBox but keep the search button? I haven't checked if its possible yet.

I'm thinking about adding a feature to save a calculated signal, but haven't implemented it yet, it could solve this issue although combining the two would be ideal in my opinion.

**Comment by danielhrisca** [comment]
With the new code you can use any channel from the MDF file, not just the ones from the Plot window
